packages/browser.smash

```diff
@@ -148,6 +148,20 @@
         }
     }
 
+    * Click each {{element}} {
+        const list = element.split(",");
+        const elems = await $$(list);
```

Comment:
No this isn't right. You had it right before. You shouldn't take a comma-separated list (among many things, the comma is a special character in an EF and this will interfere with that). You just `$$(element)` and do the `forEach()` below.

Reply:
Ok my bad